### ISSUE TITLE
Obtain currency from wallet

### ISSUE THREAD

**Issue by tselit** [interledger/web-monetization-extension#113]
### CONTEXT
Prerequisites: Assume that a WM user has already done the following:
- obtained an OP-enabled digital wallet with a wallet address.
- installed the WM extension
- loaded the extension's public key onto their digital wallet (to authenticate the extension to the user's wallet)

**Requirements**
When a user opens the extension's settings page **before** connecting their wallet to the extension, then load the form with the:
   - wallet address text enabled (ready for user input)
   - amount field visible **but** disabled (no user input)
   - radio-button for 'recurring monthly' disabled (user cannot select or focus on field)
   - radio-button for 'single-use amount' disabled (user cannot select or focus on field)
   - connect button disabled

Once a user enters the wallet address, the extension should:
   - request the user's wallet's currency 
   - enable the amount field for input & use the currency of the wallet 
   - enable the two radio-buttons for 'recurring monthly' and 'single-use amount' selection
   - enable the connect button

Once the user connects their wallet, the currency of the wallet must be correct for all forms:
- On the 'disconnect wallet' Settings page, the Remaining balance
- On the page when monetization is active, the value that a user can enter to make a once-off payment/tip

**Comment by raducristianpopa** [comment]
Closed by #169.
